spec/index.bs

```diff
@@ -4894,18 +4894,28 @@ must be well nested.
 <dl dfn-type=method dfn-for=GPUCommandEncoder>
     : <dfn>writeTimestamp(querySet, queryIndex)</dfn>
     ::
+        Writes a timestamp value into |querySet| when all previous commands have completed executing.
 
         <div algorithm=GPUCommandEncoder.writeTimestamp>
             **Called on:** {{GPUCommandEncoder}} this.
 
             **Arguments:**
             <pre class=argumentdef for="GPUCommandEncoder/writeTimestamp(querySet, queryIndex)">
-                querySet:
-                queryIndex:
+                querySet: The query set that will store the timestamp values.
+                queryIndex: The index of the query in the query set.
             </pre>
 
             **Returns:** {{undefined}}
 
+            If any of the following conditions are unsatisfied, generate a validation error and stop.
+            <div class=validusage>
+                - |this|.{{GPUCommandEncoder/[[state]]}} is {{encoder state/open}}.
+                - |querySet| is [$valid to use with$] |this|.
+                - |querySet|.{{GPUQuerySet/[[type]]}} is {{GPUQueryType/timestamp}}.
```

Comment:
I think we also need to check that the querySet isn't destroyed.

Reply:
This is covered by "valid to use with".

Reply:
Yes.  "valid to use with" validates that the querySet is not destoryed.